<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_email"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_email">
         <property name="text">
          <string>&amp;E-Mail</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_email</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_first_name"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_fax"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_telephone">
         <property name="text">
          <string>&amp;Telefon</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_telephone</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_first_name">
         <property name="text">
          <string>&amp;Vorname</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_first_name</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_telephone"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_last_name">
         <property name="text">
          <string>&amp;Nachname</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_last_name</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_last_name"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_fax">
         <property name="text">
          <string>&amp;Fax</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_fax</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_mobile">
         <property name="text">
          <string>&amp;Mobil</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_mobile</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLineEdit" name="lineEdit_mobile"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Adressse</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_street">
          <property name="text">
           <string>&amp;Straße</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_street</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_street"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_house_number">
          <property name="text">
           <string>&amp;Hausnummer</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_house_number</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_house_number"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_city">
          <property name="text">
           <string>&amp;Stadt</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_city</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_city"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_state">
          <property name="text">
           <string>&amp;Bundesland</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_state</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_state"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_zipcode">
          <property name="text">
           <string>&amp;Postleitzahl</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_zipcode</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_zipcode"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_country">
          <property name="text">
           <string>&amp;Land</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_country</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_country"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Person löschen</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_uid">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>5</pointsize>
      </font>
     </property>
     <property name="text">
      <string>uid: -</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_first_name</tabstop>
  <tabstop>lineEdit_last_name</tabstop>
  <tabstop>lineEdit_telephone</tabstop>
  <tabstop>lineEdit_mobile</tabstop>
  <tabstop>lineEdit_fax</tabstop>
  <tabstop>lineEdit_email</tabstop>
  <tabstop>lineEdit_street</tabstop>
  <tabstop>lineEdit_house_number</tabstop>
  <tabstop>lineEdit_city</tabstop>
  <tabstop>lineEdit_state</tabstop>
  <tabstop>lineEdit_zipcode</tabstop>
  <tabstop>lineEdit_country</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
